### ISSUE TITLE
Mal for hvordan saksdokument opprettes i 360

### ISSUE THREAD

**Issue by maccyber** [telemark/tfk-arkiv-metadatagenerator#4]
``` js
{
    "GetDocumentsResult": {
        "Successful": true,
        "Documents": {
            "DocumentResult": [
                {
                    "AccessCodeCode": "13",
                    "AccessCodeDescription": "OFFL§13 Taushetsplikt",
                    "CaseNumber": "14/01320",
                    "Contacts": {
                        "DocumentContactResult": [
                            {
                                "ReferenceNumber": "874568082",
                                "Role": "Avsender",
                                "SearchName": "Porsgrunn videregående skole"
                            },
                            {
                                "ReferenceNumber": "30129938600",
                                "Role": "Mottaker",
                                "SearchName": "Berit Oline Skau"
                            },
                            {
                                "ReferenceNumber": "19116544421",
                                "Role": "Mottaker",
                                "SearchName": "Donald Duck"
                            }
                        ]
                    },
                    "DocumentDate": "2016-01-20T00:00:00.000Z",
                    "DocumentNumber": "14/01320-87",
                    "Files": {},
                    "JournalDate": "2016-01-20T00:00:00.000Z",
                    "Paragraph": "Offl §13 jfr Fvl §13",
                    "Recno": 348025,
                    "ResponsibleEnterpriseName": "Porsgrunn videregående skole",
                    "ResponsiblePersonName": "",
                    "StatusCode": "J",
                    "StatusDescription": "Journalført",
                    "Title": "Varsel - orden - Berit Oline Skau - 1TIB - Porsgrunn videregående skole - 2016 - 1. termin"
                }
            ]
        }
    }
}
```

**Comment by maccyber** [comment]
@vavaen 
Er det nødvendig at "1. termin" blir med i tittel på saksdokumentet?

**Comment by zrrrzzt** [comment]
Et varsel gjelder 1. termin, 2. termin eller standpunkt. Skal man kunne ta ut rapporter på periode må den med.
Men, tittel her bør vel kommer fra [tfk-saksbehandling-elev-varsel](https://github.com/telemark/tfk-saksbehandling-elev-varsel) og ikke bygges opp av metadatageneratoren?

**Comment by maccyber** [comment]
Ja, [tfk-saksbehandling-elev-varsel](https://github.com/telemark/tfk-saksbehandling-elev-varsel) er vel den riktige plassen for den logikken.

Kan da flytte denne for tittel: [louie-create-title.js](https://github.com/telemark/tfk-arkiv-metadatagenerator/blob/master/lib/sharedFunctions/louie-create-title.js)

Er da skole-tilgangsgrupper også en ting for saksbehandleren?
ref: [student-access-group.js](https://github.com/telemark/tfk-arkiv-metadatagenerator/blob/master/lib/sharedFunctions/school-student-access-group.js)

**Comment by zrrrzzt** [comment]
Tittel kan flyttes. 
Tilgangsgrupper er jeg mer usikker på. Magefølelsen sier det er mer et arkiv-ansvar... kanskje. Kan fort ende opp med at alt av metadatagenerering flyttes ut i saksbehandlerene og det er vel neppe meningen.

**Comment by vavaen** [comment]
Hei Jonas,

Og jeg kobler på Elin her:

Jonas: Antar du mener saksdokumenttittel til 360, og ikke i selve brevet til eleven.

Hva sier du, Elin? Vil terminopplysningen forenkle søkingen for de videregående skolene?

Mvh Vanja

Fra: Jonas Enge [mailto:notifications@github.com]
Sendt: 20. januar 2016 23:26
Til: telemark/tfk-arkiv-metadatagenerator tfk-arkiv-metadatagenerator@noreply.github.com
Kopi: Vanja Valjord Christensen Vanja.Valjord.Christensen@t-fk.no
Emne: Re: [tfk-arkiv-metadatagenerator] Mal for hvordan saksdokument opprettes i 360 (#4)

@vavaenhttps://github.com/vavaen
Er det nødvendig at "1. termin" blir med i tittel på saksdokumentet?

**Comment by vavaen** [comment]
Er enig med Geir her.
Vanja

Fra: Geir Gåsodden [mailto:notifications@github.com]
Sendt: 21. januar 2016 00:31
Til: telemark/tfk-arkiv-metadatagenerator tfk-arkiv-metadatagenerator@noreply.github.com
Kopi: Vanja Valjord Christensen Vanja.Valjord.Christensen@t-fk.no
Emne: Re: [tfk-arkiv-metadatagenerator] Mal for hvordan saksdokument opprettes i 360 (#4)

Et varsel gjelder 1. termin, 2. termin eller standpunkt. Skal man kunne ta ut rapporter på periode må den med.
Men, tittel her bør vel kommer fra tfk-saksbehandling-elev-varselhttps://github.com/telemark/tfk-saksbehandling-elev-varsel og ikke bygges opp av metadatageneratoren?

**Comment by vavaen** [comment]
Nei, skole-tilgangsgruppen trenger ikke i tittelen, da jeg antar/forventer at skole-navnet er der?
Vanja

Fra: Jonas Enge [mailto:notifications@github.com]
Sendt: 21. januar 2016 00:44
Til: telemark/tfk-arkiv-metadatagenerator tfk-arkiv-metadatagenerator@noreply.github.com
Kopi: Vanja Valjord Christensen Vanja.Valjord.Christensen@t-fk.no
Emne: Re: [tfk-arkiv-metadatagenerator] Mal for hvordan saksdokument opprettes i 360 (#4)

Ja, tfk-saksbehandling-elev-varselhttps://github.com/telemark/tfk-saksbehandling-elev-varsel er vel den riktige plassen for den logikken.

Kan da flytte denne for tittel: louie-create-title.jshttps://github.com/telemark/tfk-arkiv-metadatagenerator/blob/master/lib/sharedFunctions/louie-create-title.js

Er da skole-tilgangsgrupper også en ting for saksbehandleren?
ref: student-access-group.jshttps://github.com/telemark/tfk-arkiv-metadatagenerator/blob/master/lib/sharedFunctions/school-student-access-group.js
